app-modules/service-management/src/Policies/ServiceRequestPolicy.php

```diff
@@ -96,6 +96,14 @@ public function view(Authenticatable $authenticatable, ServiceRequest $serviceRe
 
     public function create(Authenticatable $authenticatable): Response
     {
+        if (! auth()->user()->hasRole('authorization.super_admin')) {
+            $team = auth()->user()->teams()->first();
+
+            if (! $team?->managableServiceRequestTypes()->exists() && ! $team?->auditableServiceRequestTypes()->exists()) {
+                return Response::deny("You don't have permission to view this service request because you're not an auditor or manager.");
```

Comment:
```suggestion
                return Response::deny("You don't have permission to create service requests because you're not an auditor or manager of any service request types.");
```